src/blocks/gallery-masonry/edit.js

```diff
@@ -285,7 +284,7 @@ function GalleryEdit( props ) {
 			)
 			: innerBlockImages;
 
-		const newCaptions = selectedImages?.reduce( ( previous, image ) => {
+		const newCaptions = Array.from( selectedImages )?.reduce( ( previous, image ) => {
```

Comment:
Is the `?` still necessary? If `selectedImages` is `null` or `undefined`, `Array.from` will fail, right? If it is the empty string, `Array.from` will return `[]` which does not need the `?`

Reply:
Good call. There would be no optional just a fail. This is superfluous and I will remove it.